### ISSUE TITLE
Make a small correction to sumstats files for GCST005362 and GCST012597

### ISSUE THREAD

**Issue by eks-ebi** [EBISPOT/goci#589]

**Comment by eks-ebi** [comment]
@jdhayhurst This is not urgent, but could you help me with this when you have a free moment? It's presumably fairly straightforward, I just don't know exactly how to pinpoint a specific row and edit within a gzipped file.

Also wanted to check if this edit would have any flow-on effects other than changing the md5sum?

**Comment by eks-ebi** [comment]
@jdhayhurst thanks! I think it would be quicker to do the `sed` option to edit the files in situ on the ftp if that sounds ok. I've just used `zgrep` to confirm that these rows are the only instances '55512437\tA' and '157521024\tA' in both files, so there shouldn't be any off-target changes.

**Comment by eks-ebi** [comment]
Should `sed` work on a compressed file, or does it need to be unzipped first?

I did some reading on `sed` and tried this command on my own copy of one of the files (added `-n` and `/p` to print only the replaced lines), but it doesn't seem to change anything:

`sed -n 's/55512437\tA/55512437\t/p' GCST005362_buildGRCh37.tsv.gz`

**Comment by jdhayhurst** [comment]
yes, it needs to be uncompressed first. If you want to make an in situ edit use `-i`. Ensure you have a backup of the original before doing that.

**Comment by eks-ebi** [comment]
@jdhayhurst I've finally made these edits and revalidated the edited files. I have them in my "homes". What is the best way to replace the old files without messing up the harmonisation etc.?

**Comment by jdhayhurst** [comment]
released. the harmonisation pipeline had already completed for these, so I deleted the harmonised data and requeued it with the new files you provided
